integration_tests/__tests__/failures-test.js

```diff
@@ -17,30 +17,6 @@ const normalizeDots = text => text.replace(/\.{1,}$/gm, '.');
 
 skipOnWindows.suite();
 
-const cleanupStackTrace = stderr => {
```

Comment:
i was confused by this message in the CI. so i removed it.
> This function is used to remove inconsistent stack traces between
 -      jest-jasmine2 and jest-circus. If you see this error, that means the
 -      stack traces are no longer inconsistent and this function can be
 -      safely removed.

updated.